templates/environment/cf.yml

```
@@ -261,8 +212,8 @@ Resources:
       Protocol: HTTPS
 
   CloudformationExecutionRole:
-    Type: AWS::IAM::Role
-    DependsOn: VPC
+    Type: AWS::IAM::Role{{if not .ImportVpc}}
+    DependsOn: VPC{{end}}
```

nit: Can we leverage the the minus character "-" to make our templates easier to read (https://golang.org/pkg/text/template/#hdr-Text_and_spaces)?
```suggestion
    Type: AWS::IAM::Role
    {{- if not .ImportVpc}}
    DependsOn: VPC
    {{- end}}
```